tests/scripts/analyze_outcomes.py

```diff
@@ -302,6 +302,28 @@ def do_analyze_driver_vs_reference(outcome_file, args):
                     # case above.
                     ('Key ASN1 (OneAsymmetricKey X25519, doesn\'t match masking '
                      'requirements, from RFC8410 Appendix A but made into version 0)'),
+                    # When PK_PARSE_C and ECP_C are defined then PK_PARSE_EC_COMPRESSED
+                    # is automatically enabled in build_info.h (backward compatibility)
+                    # even if it is disabled in config_psa_crypto_no_ecp_at_all(). As a
+                    # consequence compressed points are supported in the reference
+                    # component but not in the accelerated one, so they should be skipped
+                    # while checking driver's coverage.
```

Comment:
Ah, thanks for the comment. When I saw the message in the list of commits I was like "??!" but it makes sense now.

Reply:
Yes, I was a bit too lazy to duplicate the full description also in the commit's text, but I thought that the comment in the code was a better place for anyone getting to this point in the future :)

Reply:
Sure! And anyway I was only looking at the summary, one line per commit, so it wouldn't have helped. And the suspense wasn't very long, just the time to review the previous 3 commits first ;)